VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -38,7 +38,7 @@
  */
 public class MainApplication extends MultiDexApplication {
     //FIXME - temporary solution
-    public static final boolean Q4_ASSESSMENT_FLAG = false;
+    public static final boolean Q4_ASSESSMENT_FLAG = true;
```

Comment:
it would be better if this went through the config system so this won't keep happening